R/base64.R

```diff
@@ -10,7 +10,15 @@ process_html_res <- function(html, reg, processor) {
       character(1)
     )
   }
-  html <- stringr::str_replace_all(html, reg, process_img_src)
+
+  # base R alternative to stringr::str_replace_all() with a function in
+  # replacement
+  mtchs <- regmatches(html, gregexpr(reg, html, perl = TRUE))[[1]]
+  if (length(mtchs) > 0) {
+    for (i in rev(mtchs)) {
+      html <- gsub(i, process_img_src(i), html, fixed = TRUE)
+    }
+  }
```

Comment:
You can use `regmatches<-` here to process the matches (I'll do it by myself in a minute).

Reply:
Thanks for the tip!